rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -471,14 +471,14 @@ def centre_crop(*inputs, bytes_per_sample_hint=0, crop=[100, 100], crop_d=1, cro
     #Set Seed
     b.setSeed(seed)
     # pybind call arguments
-    kwargs_pybind = {"input_image0": inputs[0], "crop_width":crop_width, "crop_height":crop_height, "crop_depth":crop_depth,
-                     "is_output": False}
+    kwargs_pybind = {"input_image0": inputs[0], "crop_width":crop_width, "crop_height":crop_height, 
+                     "is_output": False, "output_layout": output_layout, "output_dtype" :output_dtype}
```

Comment:
Add space after : and remove space before : to match other files

Reply:
done